qiskit/transpiler/passes/analysis/count_ops_longest_path.py

```diff
@@ -10,13 +10,13 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Count the operations on the longest path in a DAGcircuit."""
+"""Count the operations on the longest path in a :class:`.DAGCircuit`."""
```

Comment:
No need to change this PR, but don't worry about these module-level docstrings; the vast majority of them aren't built into the docs and only exist because of an overzealous pylint rule (which we could _maybe_ turn off).  The ones that are built into the docs are really pretty obviously intended for public reading - they're long and sectioned.

Reply:
reverted in 62f391ecadc92fd7d9856eb7818c25f30dc08e1a